blob/azureblob/example_test.go

```diff
@@ -25,74 +29,215 @@ import (
 
 func Example() {
 	ctx := context.Background()
-
-	accountName, accountKey := "", ""
+	// Set a fake key which must be base64 encoded. The Azure Storage Access Keys are already base64 encoded.
+	accountName, accountKey := "gocloudblobtests", base64.StdEncoding.EncodeToString([]byte("FAKECREDS"))
 	bucketName := "my-bucket"
-	key := "my-file.json"
 
-	// Initializes an azblob.ServiceURL which represents the Azure Storage Account using Shared Key authorization
-	serviceURL, _ := azureblob.ServiceURLFromAccountKey(accountName, accountKey)
+	// Initializes an azblob.ServiceURL which represents the Azure Storage Account using Shared Key authorization.
+	serviceURL, err := azureblob.ServiceURLFromAccountKey(accountName, accountKey)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+
+	// Credential represents the authorizer for SignedURL.
+	creds, err := azblob.NewSharedKeyCredential(accountName, accountKey)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
 
-	// Credential represents the authorizer for SignedURL
-	creds, _ := azblob.NewSharedKeyCredential(accountName, accountKey)
+	// Override the default retry policy so calls can return promptly for this test. Please review the timeout guidelines and set accordingly.
+	// See https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-blob-service-operations for more information.
+	opts := azblob.PipelineOptions{
+		Retry: azblob.RetryOptions{
+			Policy:        azblob.RetryPolicyFixed,
+			TryTimeout:    10 * time.Second,
```

Comment:
Does this mean the test takes 10 seconds to run? If so, please make it much shorter (here and below).

Reply:
TryTimeout is the HttpClient timeout; I will experiment and see what works best.

Reply:
Essentially, TryTimeout needs to be anything but zero. Zero triggers the default wait time which is 30 seconds. In the latest commit I set it to 5; however, the tests run under 1 seconds.